packages/core/src/errorCollection.ts

```diff
@@ -118,26 +150,39 @@ export function stopRuntimeErrorTracking() {
   ;(report.unsubscribe as (handler: Handler) => void)(traceKitReportHandler)
 }
 
-export function formatRuntimeError(stackTrace: StackTrace, errorObject: any) {
-  let message: string
-  let stack: string
-  if (stackTrace.message === undefined && !(errorObject instanceof Error)) {
-    message = `Uncaught ${jsonStringify(errorObject)}`
-    stack = 'No stack, consider using an instance of Error'
-  } else {
-    message = stackTrace.message || 'Empty message'
-    stack = toStackTraceString(stackTrace)
+export function makeAddError(errorObservable: ErrorObservable) {
+  return (addedError: AddedError, savedGlobalContext?: Context) => {
+    const stackTrace = addedError.error instanceof Error ? computeStackTrace(addedError.error) : undefined
+    const { message, stack, kind } = formatUnknownError(stackTrace, addedError.error, 'Captured')
```

Comment:
I used 'Captured' here, but I'm not convinced about it. I don't really like 'Added', but I'm open to alternatives.